packages/chakra-ui/src/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Heading } from '@chakra-ui/react'
+import { List, ListIcon, ListItem } from '@chakra-ui/react'
+import { ErrorListProps } from '@rjsf/core'
+import React from 'react'
+
+const ErrorList = ({ errors }: ErrorListProps) => (
+  <Box mb={2} p={2}>
+    <Heading as="h6" size="md" fontWeight="500" color="red.600">Errors</Heading>
```

Comment:
Can you add a bit more space below "Error list"? Looks a bit cramped now.

![image](https://user-images.githubusercontent.com/1689183/111926870-f2dfc700-8a84-11eb-9224-41f6b02cfd0b.png)